src/main/java/com/carefusion/carefusion_backend/model/entity/Patient.java

```diff
@@ -21,6 +21,45 @@ public class Patient extends BaseSoftDeleteModel {
   @Column(name = "person_id")
   private Long personId;
 
+  @Column(name = "primary_phone")
+  private String primaryPhone;
+
+  @Column(name = "birthdate")
+  private String birthdate;
```

Comment:
birthday column 的型別是ZonedDateTime
```suggestion
  private ZonedDateTime birthdate;
```

Reply:
這個沒改到~ 不過我看一下這個應該是要改成LocalDate
```suggestion
  private LocalDate birthdate;
```

Reply:
不過這個我想請問一下
這兩個type都是要另外import的對嗎？
因為我看他有給我這個提示

Reply:
沒錯～是要import才能用的